### PR TITLE
Add value input to tiller installer

### PR DESCRIPTION
**Pull request by MarcelMue**
Required to augment the tiller deployment and add priorityclasses, tolerations etc pp.

### CODE REVIEW

File: helmclient.go [hunk 1]
```diff
@@ -286,6 +295,7 @@ func (c *Client) EnsureTillerInstalled(ctx context.Context) error {
 		MaxHistory:                   defaultMaxHistory,
 		Namespace:                    c.tillerNamespace,
 		ServiceAccount:               tillerPodName,
+		Values:                       values,
```
Comment (MarcelMue) [inline helmclient.go]: This is all that changes, very straight forward just passing this through to the helm API.

File: helmclient.go [hunk 2]
```diff
@@ -153,6 +153,15 @@ func (c *Client) DeleteRelease(ctx context.Context, releaseName string, options
 // As a first step, it checks if Tiller is already ready, in which case it
 // returns early.
 func (c *Client) EnsureTillerInstalled(ctx context.Context) error {
+	return c.EnsureTillerInstalledWithValues(ctx, []string{})
```
Comment (MarcelMue) [inline helmclient.go]: Works fine and preserves the behaviour.

**Comment by MarcelMue** [comment]
@rossf7 `chart-operator` for now, so we get the correct priority class and so on in tenant clusters.

**Comment by rossf7** [comment]
Ack. But tiller in tenant clusters is first created by the tiller resource in cluster-operator. This is for bootstrapping because we install chart-operator with helm.

https://github.com/giantswarm/cluster-operator/blob/4be49f490f2548ffe82c6ec0f1c730265cf9eb86/pkg/v13/resource/tiller/resource.go#L68

**Comment by MarcelMue** [comment]
@rossf7 Argh I see - I was starting to wonder why it wasn't working from the chart-operator already ...
I will add my changes to cluster-operator

**Comment by rossf7** [comment]
Cool I think its good to have it in both.

**Pull request merged**